Makefile

```diff
@@ -14,6 +14,7 @@ GIT_TREE_STATE=$(shell if [ -z "`git status --porcelain`" ]; then echo "clean" ;
 DOCKER_PUSH?=false
 IMAGE_NAMESPACE?=argoproj
 VERSION?=latest
+BASE_VERSION:=latest
```

Comment:
To clarify:

`BASE_VERSION` is the original image tag in the manifests.
`VERSION` is the new image tag that could be passed in during image build.